### DISCUSSION TITLE
Dumb Question- how do I install this to a repo?

### DISCUSSION THREAD

**Discussion by arn5891** [Rapptz/discord.py#9489] [Q&A] [answered]
I can't really figure out how to install modules in GitHub. Discord.py doesn't show up in the packages tab, and I'm not sure how to access bash. Do I have to download something and upload it to my repo?

**Comment by No767** [comment, marked as answer]
Seems to me that you just need to upload your requirements.txt or anything similar to the repo, and github will automatically detect it. 

You don't install modules in github. Github automatically detects and tracks the dependencies as needed

Edit: If you are referring to the GitHub packages registry (such as ghcr, etc), then you don't have to worry about it. It won't detect it as a pypi package unless you build and push it off to github's pypi registry, which i dont know why you would want to do that anyways

**Reply by arn5891** [reply]
Thanks, that fixed the problem.
